src/exception.cc

```diff
@@ -301,10 +301,69 @@ namespace std
 
 using namespace ABI_NAMESPACE;
 
+/**
+ * Callback function used with _Unwind_Backtrace().
+ *
+ * Prints a stack trace.  Used only for debugging help.
+ *
+ * Note: As of FreeBSD 8.1, dladd() still doesn't work properly, so this only
```

Comment:
Typo moved from the old location (missing r at the end of dladdr)

Reply:
Fixed, thanks.